compiler/test-resources/repl/i5855

```diff
@@ -0,0 +1,2 @@
+scala> def f given Int = 1
```

Comment:
I suggest having all the definition tests in `dotty/compiler/test-resources/repl/defs`